src/forecast.py

```diff
@@ -137,11 +151,39 @@ def forecast_page():
             regime="day-after",
         )
 
-        pvlive_data = {}
-        pvlive_data["PVLive Initial estimate"] = [GSPYield.from_orm(f) for f in pvlive_inday]
-        pvlive_data["PVLive Updated estimate"] = [GSPYield.from_orm(f) for f in pvlive_dayafter]
+        pvlive_gsp_sum_inday = get_gsp_yield_sum(
+            session=session,
+            gsp_ids=list(range(1, 318)),
+            start_datetime_utc=start_datetime,
+            end_datetime_utc=end_datetime,
+            regime="in-day",
+        )
 
-    # make plot
+        pvlive_gsp_sum_dayafter = get_gsp_yield_sum(
+            session=session,
+            gsp_ids=list(range(1, 318)),
+            start_datetime_utc=start_datetime,
+            end_datetime_utc=end_datetime,
+            regime="day-after",
+        )
+
+        pvlive_data = {}
+        pvlive_data["PVLive Initial Estimate"] = [
+            GSPYield.from_orm(f) for f in pvlive_inday
+        ]
+        pvlive_data["PVLive Updated Estimate"] = [
+            GSPYield.from_orm(f) for f in pvlive_dayafter
+        ]
+
+        pvlive_gsp_sum_data = {}
```

Comment:
should we only load this if gsp_id==0

Reply:
probably. it doesn't really make sense to load this data every time.